pygmt/datasets/load_remote_dataset.py

```diff
@@ -41,33 +41,33 @@ class GMTRemoteDataset(NamedTuple):
 
     Attributes
     ----------
-    title : str
-        The title of the dataset, used in error messages.
     name : str
-        The name assigned as an attribute to the DataArray.
-    long_name : str
+        Always ``"z"``.
+    long_name : str, None
         The long name assigned as an attribute to the DataArray.
-    units : str
+    description : str
+       The name assigned as an attribute to the DataArray.
+    units : str, None
         The units of the values in the DataArray.
     resolutions : dict
         Dictionary of available resolution as keys and Resolution objects as values.
     extra_attributes : dict
         A dictionary of extra or unique attributes of the dataset.
     """
 
-    title: str
     name: str
-    long_name: str
+    long_name: str | None
+    description: str
     units: str | None
     resolutions: dict[str, Resolution]
     extra_attributes: dict
 
 
 datasets = {
     "earth_age": GMTRemoteDataset(
-        title="seafloor age",
-        name="seafloor_age",
-        long_name="age of seafloor crust",
+        name="z",
+        long_name="age_Myr",
```

Comment:
With PR #3120 merged, the grid name defaults to `z` and `long_name` defaults to `ages (Myr)`. So we can remove the `name` and `long_name` attributes.
```suggestion
```